drivers/SmartThings/zigbee-switch/src/test/test_aqara_smart_plug.lua

```diff
@@ -0,0 +1,278 @@
+local PRIVATE_ATTRIBUTE_ID = 0x0009
+
+local PREF_CLUSTER_ID = 0xFCC0
+local PREF_MAX_POWER_ATTR_ID = 0x020B
+local PREF_RESTORE_STATE_ATTR_ID = 0x0201
+
+local LAST_REPORT_TIME = "LAST_REPORT_TIME"
+
+local mock_device = test.mock_device.build_test_zigbee_device(
+  {
+    profile = t_utils.get_profile_definition("switch-power-energy-consumption-report-aqara.yml"),
+    fingerprinted_endpoint_id = 0x01,
+    zigbee_endpoints = {
+      [1] = {
+        id = 1,
+        manufacturer = "LUMI",
+        model = "lumi.plug.maeu01",
+        server_clusters = { SimpleMetering.ID, ElectricalMeasurement.ID }
+      }
+    }
+  }
+)
+
+zigbee_test_utils.prepare_zigbee_env_info()
+local function test_init()
+  test.mock_device.add_test_device(mock_device)
+  zigbee_test_utils.init_noop_health_check_timer()
+end
+
+test.set_test_init_function(test_init)
+
+test.register_coroutine_test(
+  "Handle added lifecycle",
+  function()
+    test.socket.device_lifecycle:__queue_receive({ mock_device.id, "added" })
+    test.socket.zigbee:__expect_send(
+      {
+        mock_device.id,
+        Groups.server.commands.RemoveAllGroups(mock_device)
+      }
+    )
+
+    test.socket.capability:__expect_send(
+      mock_device:generate_test_message("main", capabilities.switch.switch.off())
+    )
+    test.socket.capability:__expect_send(
+      mock_device:generate_test_message("main", capabilities.energyMeter.energy({ value = 0, unit = "kWh" }))
+    )
+    test.socket.capability:__expect_send(
+      mock_device:generate_test_message("main", capabilities.powerMeter.power({ value = 0, unit = "W" }))
+    )
+
+    test.socket.zigbee:__expect_send(
+      {
+        mock_device.id,
+        cluster_base.write_manufacturer_specific_attribute(mock_device, PRIVATE_CLUSTER_ID, PRIVATE_ATTRIBUTE_ID,
+          MFG_CODE, data_types.Uint8, 1)
+      }
+    )
+
+    test.socket.zigbee:__expect_send({ mock_device.id,
+      cluster_base.write_manufacturer_specific_attribute(mock_device, PREF_CLUSTER_ID, PREF_MAX_POWER_ATTR_ID,
+        MFG_CODE,
+        data_types.SinglePrecisionFloat,
+        SinglePrecisionFloat(0, 11, 0.123046875)) })
+    test.socket.zigbee:__expect_send({ mock_device.id,
+      cluster_base.write_manufacturer_specific_attribute(mock_device, PREF_CLUSTER_ID,
+        PREF_RESTORE_STATE_ATTR_ID, MFG_CODE,
+        data_types.Boolean,
+        false) })
+  end
+)
+
+test.register_coroutine_test(
+  "Handle doConfigure lifecycle",
+  function()
+    test.socket.device_lifecycle:__queue_receive({ mock_device.id, "doConfigure" })
+    test.socket.zigbee:__set_channel_ordering("relaxed")
+
+    test.socket.zigbee:__expect_send({
+      mock_device.id,
+      zigbee_test_utils.build_bind_request(mock_device,
+        zigbee_test_utils.mock_hub_eui,
+        SimpleMetering.ID)
+    })
+    test.socket.zigbee:__expect_send({
+      mock_device.id,
+      SimpleMetering.attributes.InstantaneousDemand:configure_reporting(mock_device, 1, 3600, 1)
```

Comment:
The defaults have been changed here in v45 which is why this test is failing.

Please test with the latest artifact and update this test accordingly: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/releases/tag/apiv2_45_1

Reply:
```suggestion
      SimpleMetering.attributes.InstantaneousDemand:configure_reporting(mock_device, 1, 3600, 5)
```

Reply:
I updated. Thank you.